core/src/main/java/org/apache/iceberg/BaseReplacePartitions.java

```diff
@@ -27,6 +27,7 @@
     extends MergingSnapshotProducer<ReplacePartitions> implements ReplacePartitions {
   BaseReplacePartitions(String tableName, TableOperations ops) {
     super(tableName, ops);
+    set("replace-partitions", "true");
```

Comment:
can we make `replace-partitions` a static variable in `SnaphotSummary.java`?

Reply:
Done.